### ISSUE TITLE
Install fails: file INSTALL cannot find

### ISSUE THREAD

**Issue by yurivict** [clangen/musikcube#206]
```
-- Installing: /usr/ports/multimedia/musikcube/work/stage/usr/local/share/musikcube/locales/ru_RU.json
-- Installing: /usr/ports/multimedia/musikcube/work/stage/usr/local/share/musikcube/libmusikcore.so
-- Installing: /usr/ports/multimedia/musikcube/work/stage/usr/local/share/musikcube/musikcube
CMake Error at cmake_install.cmake:119 (file):
  file INSTALL cannot find
  "/usr/ports/multimedia/musikcube/work/musikcube-0.51.0/src/musikcube/musikcube".
```

**Comment by clangen** [comment]
Hmm, not sure what's going on here. I've never had an `INSTALL` file in the repo. Building from source then doing a `make install` on FreeBSD (which I believe is the OS you're using) seems fine to me. 

Can you provide some more context?

**Comment by yurivict** [comment]
It might be the same problem: https://stackoverflow.com/questions/25272866/cmake-make-install-fails-with-file-install-cannot-find-myfile

**Comment by clangen** [comment]
OK, so, I'm still not able to repro, but this seems to be a problem with the wrapper shell script that starts the actual `musikcube` executable. This script is generated at build time. I seem to remember you also had problems with `cmake` because it was using `ninja` instead of standard unix makefiles. I have a feeling that may have been related, which caused the input `musikcube.in` to not get processed/generated properly. 

In investigating this I also discovered a slight "out of the box incompatibility" with the wrapper script on FreeBSD; it seems FreeBSD does not ship with `bash`, only `sh`, which means these scripts may not work properly for some users. 

I updated the scripts to use `sh` instead, and pushed some other minor changes related to BSD compatibility.

If you get a chance, can you try pulling the latest from git and rebuilding?

- cd ~/path/to/musikcube/
- git pull
- ./clean-nix.sh
- cmake -G "Unix Makefiles" .
- make
- make install

I tried this on a clean FreeBSD install and it seems to work fine.

**Comment by yurivict** [comment]
Now I run w/out ninja.

To reproduce:
1. cd /usr/ports
2. wget https://people.freebsd.org/~yuri/musikcube.shar
3. sh musikcube.shar
4. cd multimedia/musikcube
5. make

**Comment by clangen** [comment]
@yurivict thanks for the repro steps! You were correct in your original suggestion, the fix required using `CMAKE_CURRENT_BINARY_DIR` as suggested in that SO thread. I've pushed changes to the `master` branch that should get you up and running... let me know if it fixes things on your end!

Also, I noticed the license specified in `musikcube.shar` is incorrect -- musikcube uses a standard 3-clause BSD, not GPLv2! :)
